components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -284,22 +284,27 @@ function RepositoryNotFoundView(p: { error: StartWorkspaceError }) {
 
       if (!updatedRecently) {
         setStatusMessage(<div className="mt-2 flex flex-col space-y-8">
-          <p className="text-base text-gray-400 w-96">Permission to access private repositories has been granted. If you are a member of '{owner}', please try to request access for Gitpod.</p>
+          <p className="text-base text-gray-400 w-96">Permission to access private repositories has been granted. If you are a member of <code>{owner}</code>, please try to request access for Gitpod.</p>
           <a className="mx-auto" href={authorizeURL}><button className="secondary">Request Access for Gitpod</button></a>
         </div>);
         return;
       }
 
       setStatusMessage(<div className="mt-2 flex flex-col space-y-8">
-        <p className="text-base text-gray-400 w-96">Your access token was updated recently. Please try again if the repository exists and Gitpod was approved for '{owner}'.</p>
+        <p className="text-base text-gray-400 w-96">Your access token was updated recently. Please try again if the repository exists and Gitpod was approved for <code>{owner}</code>.</p>
```

Comment:
**thought**: We could probably use the same `CodeText` component here as mentioned below. 💭